uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
+                    model_instance = model()
+                    session.add(model_instance)
+
+            for colname, val in row.items():
+                col = cols[colname]
+                if not val:
+                    # in a lot of cases we'll just have the empty string, so we'll just
+                    # do nothing for those cases
+                    continue
+                if isinstance(col.type, Choice):
+                    # the export has labels, and we want to convert those back into their
+                    # integer values, so let's look that up (note: we could theoretically
+                    # modify the Choice class to do this automatically in the future)
+                    label_lookup = {val: key for key, val in col.type.choices.items()}
+                    val = label_lookup[val]
+                elif isinstance(col.type, MultiChoice):
+                    # the export has labels separated by ' / ' and we want to convert that
+                    # back into a comma-separate list of integers
+                    label_lookup = {val: key for key, val in col.type.choices}
+                    vals = [label_lookup[label] for label in val.split(' / ')]
+                    val = ','.join(map(str, vals))
+                elif isinstance(col.type, UTCDateTime):
+                    # we'll need to make sure we use whatever format string we used to
+                    # export this date in the first place
+                    try:
+                        val = UTC.localize(datetime.strptime(val, date_format + ' %H:%M:%S'))
+                    except:
+                        val = UTC.localize(datetime.strptime(val, date_format))
+                elif isinstance(col.type, Date):
+                    val = datetime.strptime(val, date_format).date()
+                elif isinstance(col.type, Integer):
+                    val = int(val)
+
+                # now that we've converted val to whatever it actually needs to be, we
+                # can just set it on the model
+                setattr(model_instance, colname, val)
+
+            try:
+                session.commit()
+            except:
+                log.error('ImportError', exc_info=True)
+                session.rollback()
+                message = 'Import unsuccessful'
+
+            id_list.append(model_instance.id)
+
+        all_instances = session.query(model).filter(model.id.in_(id_list)).all() if id_list else None
+
+        if model == Attendee:
+            form_link = '../registration/form'
+        elif model == Group:
+            form_link = '../groups/form'
+        elif model == Job:
+            form_link = '../jobs/form'
+        else:
+            form_link = None
+
+        return self.csv_import(message, all_instances, form_link)
+
+    def valid_attendees(self):
+        self.export_model(selected_model='attendee')
+
+    def all_attendees(self):
+        self.export_model(selected_model='attendee')
```

Comment:
In general, this should probably be ``return self.export_model(...)`` instead of just calling it.  In this particular case I'm not sure it matters since this is a CSV export, but in other cases it definitely would and you'd need the ``return`` statement for it to work.